addons/google_calendar/tests/test_token_access.py

```diff
@@ -31,36 +30,17 @@ def setUpClass(cls):
             'groups_id': [Command.link(cls.env.ref('base.group_system').id)],
         })
 
-    def test_normal_user_should_be_able_to_reset_his_own_token(self):
-        user = self.users[0]
-        old_validity = user.google_calendar_account_id.calendar_token_validity
-
-        user.with_user(user).google_calendar_account_id._set_auth_tokens('my_new_token', 'my_new_rtoken', 3600)
-
-        self.assertEqual(user.google_calendar_account_id.calendar_rtoken, 'my_new_rtoken')
-        self.assertEqual(user.google_calendar_account_id.calendar_token, 'my_new_token')
-        self.assertNotEqual(
-            user.google_calendar_account_id.calendar_token_validity,
-            old_validity
-        )
-
-    def test_normal_user_should_not_be_able_to_reset_other_user_tokens(self):
-        user1, user2 = self.users
-
-        with self.assertRaises(AccessError):
-            user2.with_user(user1).google_calendar_account_id._set_auth_tokens(False, False, 0)
-
```

Comment:
Note: this test should be reactivated.
`test_normal_user_should_not_be_able_to_reset_other_user_tokens` is not correct anymore as now every user is able to read the token of other user.
It is quite an issue and we will discuss if we need to add groups="base.group_system" on the field definition to prevent reading and access the values with sudo when necessary. 
In all cases, these implications should be explained in the commit message.